package.json

```diff
@@ -29,67 +29,73 @@
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.24.3",
-    "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.24.3",
-    "@babel/preset-react": "^7.24.1",
+    "@babel/cli": "^8.0.0-beta.2",
+    "@babel/core": "^8.0.0-beta.2",
+    "@babel/eslint-parser": "^8.0.0-beta.2",
+    "@babel/plugin-external-helpers": "^8.0.0-beta.2",
+    "@babel/plugin-transform-class-properties": "^8.0.0-beta.2",
+    "@babel/plugin-transform-runtime": "^8.0.0-beta.2",
+    "@babel/preset-env": "^8.0.0-beta.2",
+    "@babel/preset-react": "^8.0.0-beta.2",
     "@cyclonedx/webpack-plugin": "^2.0.2",
-    "@jahia/eslint-config": "^2.1.2",
     "auto": "^11.1.2",
-    "babel-loader": "^9.1.3",
+    "babel-loader": "^10.0.0",
     "babel-plugin-lodash": "^3.3.4",
     "clean-webpack-plugin": "^4.0.0",
     "cryptiles": "^4.1.3",
-    "css-loader": "5.2.7",
+    "css-loader": "^7.1.2",
+    "eslint": "^9.38.0",
+    "eslint-config-xo": "^0.49.0",
+    "eslint-config-xo-react": "^0.28.0",
+    "eslint-plugin-import": "^2.32.0",
+    "eslint-plugin-n": "^17.23.1",
+    "eslint-plugin-promise": "^7.2.1",
+    "eslint-plugin-react": "^7.37.5",
+    "eslint-plugin-react-hooks": "^7.0.0",
     "file-loader": "^6.2.0",
-    "hoek": "^6.1.3",
-    "husky": "^9.0.11",
-    "loader-utils": "1.4.1",
+    "husky": "^9.1.7",
+    "loader-utils": "^3.3.1",
     "pom-parser": "^1.2.0",
-    "rimraf": "^5.0.5",
-    "style-loader": "2.0.0",
+    "rimraf": "^6.0.1",
+    "style-loader": "^4.0.0",
     "sync-pom-version-to-package": "^1.6.1",
-    "webpack": "^5.91.0",
-    "webpack-bundle-analyzer": "^4.10.1",
-    "webpack-cli": "^5.1.4",
+    "webpack": "^5.102.1",
+    "webpack-bundle-analyzer": "^4.10.2",
+    "webpack-cli": "^6.0.1",
     "webpack-node-externals": "^3.0.0"
   },
   "dependencies": {
-    "@date-io/moment": "^3.0.0",
-    "@elastic/react-search-ui": "^1.21.1",
-    "@elastic/react-search-ui-views": "^1.21.1",
-    "@elastic/search-ui": "^1.21.1",
+    "@date-io/moment": "^3.2.0",
+    "@elastic/react-search-ui": "^1.24.2",
+    "@elastic/react-search-ui-views": "^1.24.2",
+    "@elastic/search-ui": "^1.24.2",
     "@jahia/i18next": "^1.1.0",
     "@jahia/search-ui-jahia-connector": "^2.0.16",
-    "@jahia/test-framework": "^1.2.1",
-    "classnames": "^2.2.6",
+    "@jahia/test-framework": "^1.3.1",
+    "classnames": "^2.5.1",
+    "core-js": "^3.46.0",
     "cryptiles": "^4.1.3",
-    "es6-symbol": "^3.1.1",
-    "fuse.js": "^3.4.4",
+    "es6-symbol": "^3.1.4",
+    "fuse.js": "^7.1.0",
     "hoek": "^6.1.3",
-    "i18next": "^23.10.1",
+    "i18next": "^25.6.0",
     "i18next-xhr-backend": "^3.2.2",
     "lodash": "^4.17.21",
-    "moment": "^2.24.0",
-    "prop-types": "^15.7.2",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-i18next": "^14.1.0",
-    "react-icons": "^5.0.1",
+    "moment": "^2.30.1",
+    "prop-types": "^15.8.1",
+    "react": "^19.2.0",
```

Comment:
Are we sure this has no side effects? 19 has a lot of breaking changes and since react is declared as a singleton on the module federation, I don't know if this would update react for all modules and have any effect. I would stick with latest 18 for now if 19 is not required. https://github.com/Jahia/javascript-components/blame/master/packages/webpack-config/getModuleFederationConfig.js#L67